src/main/java/io/connect/scylladb/ClusterAddressTranslator.java

```diff
@@ -0,0 +1,66 @@
+package io.connect.scylladb;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.policies.AddressTranslator;
+import java.net.InetSocketAddress;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import org.json.*;
+
+class ClusterAddressTranslator implements AddressTranslator {
+
+    public Map<InetSocketAddress, InetSocketAddress> addressMap = new HashMap<>();
+
+    @Override
+    public void init(Cluster cluster) {
```

Comment:
What is this cluster object for?

Reply:
And, of course, what is this method for?

Reply:
Cluster object is the main entry point of the driver, for accessing to a Scylla cluster. This method initializes this address translator.